source/code/java/authentication/mfa_get_account_factors1_resp.java

```diff
@@ -0,0 +1,17 @@
+// In order to get all Factors
+FactorList factors = account.getFactors();
+
+Map<String,String> parameters = new HashMap<>(1);
+
+// In order to get SmsFactor
+parameters.put("type", "sms");
+FactorList factors = account.getFactors(parameters);
```

Comment:
Please have the code with the Map separated from the code with the criteria. The code must always be "compilable working java code". If someone copies this code it will not compile (for example, factors is defined more than once)